tests/phpunit/elementor/core/test-documents-manager.php

```diff
@@ -26,4 +23,11 @@ public function test_ajax_get_document_config__set_id_build_with_elementor() {
 			get_post_meta( $post->ID, Document::BUILT_WITH_ELEMENTOR_META_KEY, true )
 		);
 	}
+
+	public function test_get_new_post_url() {
+		$new_post_url =  Documents_Manager::get_create_new_post_url();
```

Comment:
Use it from the Plugin.

Reply:
Handled https://github.com/elementor/elementor/commit/23259ba9e457c9747a2ebf03ec3d5125c40ba9c7